keyboards/crkbd/rev1/split_scomm.c

```diff
@@ -9,6 +9,7 @@
 #include "serial.h"
 #ifdef CONSOLE_ENABLE
   #include <print.h>
+  #include <debug.h>
```

Comment:
```suggestion
  #include "print.h"
  #include "debug.h"
```

Reply:
Also, debug.h includes print.h, so shouldn't need both here, just one or the other.

Reply:
Thanks for the changes!